layout/theme.liquid

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta name="theme-color" content="">
     <link rel="canonical" href="{{ canonical_url }}">
+    <link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css" rel="stylesheet">
```

Comment:
I had this in `platter-product-gallery.liquid` section at first, but then figured that in a real case scenario we would need Tailwind across the entire site.